packages/synapse-react-client/src/components/HeaderCard.tsx

```diff
@@ -62,6 +66,11 @@ function HeaderCard({
     }
   })
 
+  const { appId } = useSynapseContext()
+  if (['standards', 'nf'].includes(String(appId))) {
```

Comment:
Testing on nf makes sense, but 'nf' should be removed from this check before merge